src/store/dashboard/awsDashboard/awsDashboardWidgets.ts

```diff
@@ -120,6 +123,7 @@ export const networkWidget: AwsDashboardWidget = {
     computedReportItem: ComputedReportItemType.cost,
     computedReportItemValue: ComputedReportItemValueType.total,
     titleKey: messages.dashboardCumulativeCostComparison,
+    chartName: 'dashboardCumulativeCostComparisonNetwork',
```

Comment:
Would you mind prefixing the chart names with "aws"; for example, "awsNetworkChart"